source/includes/_snapshots.md

```diff
@@ -281,6 +283,9 @@ Property | Default | Description
 ---------|---------|-------------
 `force`  |  false  | Specifies whether to force restoring the snapshot on a manager that already contains blueprints/deployments.
 `recreate_deployments_envs` | true | Specifies whether deployment environments should be created for restored deployments.
+`tenant_name` | true | In case of restoring a snapshot from a version prior to 4.0.0, it specifies the name of a new tenant that the snapshot will be restored into. Otherwise - not relevant.
```

Comment:
~In case of restoring a snapshot from a version prior to 4.0.0, it~
**In the case of a snapshot that was restored from a version prior to version 4.0.0, specifies...**